jazz_codeq/config/dev-config.json

```diff
@@ -3,15 +3,15 @@
 	"SONAR_PASSWORD": "{sonar_creds}",
 	"SONAR_URL": "{sonar_url}",

```

Comment:
looks like this config value was incorrectly labeled, this should be SONAR_HOSTNAME and not URL. Can you fix it so that in code where we are joining this, it is not confusing?